### PR TITLE
Allow setting deis executable.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: vars/deisLogin.groovy [hunk 1]
```diff
@@ -20,9 +24,9 @@ def call(String server, String credentials, Closure body) {
   }
 }
 
-def call(String server, String credentials) {
+def call(String server, String credentials, String deis_executable) {
   withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: credentials,
                     usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD']]) {
-    deisLogin(server, env.USERNAME, env.PASSWORD)
+    deisLogin(server, env.USERNAME, env.PASSWORD, deis_executable)
```
Comment (jgmize) [inline vars/deisLogin.groovy]: Did you mean to make the new param optional here to match the other functions where you're adding it?

Comment (jgmize) [inline vars/deisLogin.groovy]: clicked on wrong line-- meant to add this comment 3 lines above in the `def call(...)`

Comment (glogiotatidis) [inline vars/deisLogin.groovy]: Yes, nice catch, updating.

**Comment by glogiotatidis** [comment]
Updated and merging. Thanks!

**Pull request merged**